### ISSUE TITLE
Doesn't work with Postgres DB

### ISSUE THREAD

**Issue by oldskool73** [itsgoingd/clockwork#176]
Hi. Using Laravel 5.4. When `config.clockwork.storage = sql` and the datbase used is Postgres, the DB creation fails because:

* type DOUBLE does not exist.
* type MEDIUMTEXT does not exist.

Also, once those are fixed, the version being an `INTEGER` creates issue because you're currently putting `1.14.2` in there, which obviously isn't an int :)

The following statement fixes these issues (may break mySQL etc though...?) :

```
"CREATE TABLE {$this->table} (" .
				'id VARCHAR(100), ' .
				'version VARCHAR(10), ' .
				'time DOUBLE PRECISION NULL, ' .
				'method VARCHAR(10) NULL, ' .
				'uri VARCHAR(250) NULL, ' .
				'headers TEXT NULL, ' .
				'controller VARCHAR(250) NULL, ' .
				'getData TEXT NULL, ' .
				'postData TEXT NULL, ' .
				'sessionData TEXT NULL, ' .
				'cookies TEXT NULL, ' .
				'responseTime DOUBLE PRECISION NULL, ' .
				'responseStatus INTEGER NULL, ' .
				'responseDuration DOUBLE PRECISION NULL, ' .
				'databaseQueries TEXT NULL, ' .
				'databaseDuration DOUBLE PRECISION NULL, ' .
				'timelineData TEXT NULL, ' .
				'log TEXT NULL, ' .
				'routes TEXT NULL, ' .
				'emailsData TEXT NULL, ' .
				'viewsData TEXT NULL, ' .
				'userData TEXT NULL' .
			');'
```

**Comment by itsgoingd** [comment]
Hey, thanks for the heads up. I've been testing with Sqlite and MySQL only so far, I'll look into Postgres and hopefully find a way to support it without duplicating the queries.

The version column value should actually be the metadata version, not the Clockwork library version, so the type is correct, we are storing a wrong value though, oops.

**Comment by itsgoingd** [comment]
Implemented in master, please note that master can be unstable right now as I'm working on a new major release. If you'd like to backport the changes yourself, check out the linked PR, I've found and fixed some more compatibility issues.

**Comment by itsgoingd** [comment]
Now available in Clockwork 2.0.
